packages/elements/src/components/Typography/index.tsx

```diff
@@ -1,79 +1,261 @@
 import * as React from 'react'
```

Comment:
See my two comments, Can you help me to check for this file.
Sorry my computer is too laggy when scrolling down